flow-typed/npm/@react-navigation/bottom-tabs_v5.x.x.js

```diff
@@ -2,7 +2,6 @@
 // flow-typed version: 510a06cce0/@react-navigation/bottom-tabs_v5.x.x/flow_>=v0.104.x
```

Comment:
Wouldn't it be better to ignore the `flow-type/` folder? (as well as any other third-party code?)

Reply:
done